src/index.html

```diff
@@ -22,6 +22,371 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
```

Comment:
The class `is-active` does not follow BEM naming conventions. Consider using a modifier class like `nav__link--active` instead. [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) emphasize following the BEM methodology for naming CSS classes.